src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -79,15 +79,72 @@ public void cleanUp() {
   @Test
   public void doPost_loggedOut_postForbidden() throws ServletException, IOException {
     testServices.setEnvIsLoggedIn(false);
+    final int parentId = 32;
```

Comment:
remove this as it adds complexity for no reason. This variable is used one and converted to a string. Just pass the string directly.

Reply:
It was also used in the assertion to check the value was set.  Removed in favor of 2 literals